back/engines/commercial/admin_api/app/controllers/admin_api/users_controller.rb

```diff
@@ -34,8 +34,11 @@ def create
       @user.locale ||= AppConfiguration.instance.settings('core', 'locales').first
       SideFxUserService.new.before_create @user, nil
 
+      @user.confirm if [true, 'TRUE', 'true', '1', 1].include?(params[:confirm_email])
```

Comment:
The code still fails because `@user.confirm` has no effect. Its changes are cleared after the `save`.

In fact, I can't find a way to confirm the User programmatically, after trying a few things. Can you guide me in the right direction @jamesspeake ?

Reply:
This is outdated, fixed it with a rather dirty 2nd save operation instead. Seems like the user model doesn't allow on to confirm a user on creation in any other way, without skipping all model callbacks.